c10/core/impl/PyInterpreter.h

```diff
@@ -236,6 +241,20 @@ struct C10_API PyInterpreter {
     return (*layout_fn_)(this, self);
   }
 
+  __ubsan_ignore_function__ void trace_cuda_event_creation(uintptr_t) const;
+
+  __ubsan_ignore_function__ void trace_cuda_event_deletion(uintptr_t) const;
+
+  __ubsan_ignore_function__ void trace_cuda_event_record(uintptr_t, uintptr_t) const;
+
+  __ubsan_ignore_function__ void trace_cuda_event_wait(uintptr_t, uintptr_t) const;
+
+  __ubsan_ignore_function__ void trace_cuda_memory_allocation(uintptr_t) const;
+
+  __ubsan_ignore_function__ void trace_cuda_memory_deallocation(uintptr_t) const;
+
+  __ubsan_ignore_function__ void trace_cuda_stream_allocation(uintptr_t) const;
+
```

Comment:
Similarly, if you define the struct in this file, all of these could be defined inline. (Just like the previous methods are)